packages/loot-core/src/server/accounts/transaction-rules.ts

```diff
@@ -836,10 +861,12 @@ export async function updateCategoryRules(transactions) {
 
 export type TransactionForRules = TransactionEntity & {
   payee_name?: string;
+  _account?: AccountEntity;
 };
 
 export async function prepareTransactionForRules(
   trans: TransactionEntity,
+  accounts: Map<string, AccountEntity> = null,
```

Comment:
Can we change this to

```suggestion
  accounts: Map<string, AccountEntity> | null = null,
```

or

```suggestion
  accounts: Map<string, AccountEntity> = new Map(),
```